taca_ngi_pipeline/cli.py

```diff
@@ -87,6 +88,22 @@ def mosler(ctx, projectid):
             **ctx.parent.params)
         _exec_fn(d, d.deliver_project)
 
+#Castor Delivery
+# project delivery
+@deliver.command()
+@click.pass_context
+@click.argument('projectid', type=click.STRING, nargs=-1)
+def castor(ctx, projectid):
+    """ Deliver the specified projects to Bianca wharf.
+    """
+    for pid in projectid:
+        d = _deliver_castor.CastorProjectDeliverer(
+            pid,
+            **ctx.parent.params)
+        _exec_fn(d, d.deliver_project)
+
+
+
```

Comment:
For consistency, I would propose a slightly different organization of the commands, so that the milou/mosler/castor destination step is indicated on the group (?) level. I.e. so that the command will be e.g. `taca deliver project`, `taca deliver_castor project` etc. Alternatively, the destination could be specified as an argument I guess..